Source/WebKit/UIProcess/BrowsingContextGroup.cpp

```diff
@@ -44,7 +44,7 @@ BrowsingContextGroup::~BrowsingContextGroup() = default;
 
 Ref<FrameProcess> BrowsingContextGroup::ensureProcessForSite(const Site& site, WebProcessProxy& process, const WebPreferences& preferences, InjectBrowsingContextIntoProcess injectBrowsingContextIntoProcess)
 {
-    if (!site.isEmpty() && preferences.siteIsolationEnabled()) {
+    if (preferences.siteIsolationEnabled()) {
         if (RefPtr existingProcess = processForSite(site)) {
```

Comment:
Is empty / null site a valid key in the site process map?

Reply:
It is!